### PR TITLE
Feat/big movie card structure

### PR DESCRIPTION
**Pull request by Aditi-Sawardekar**
# Description: Create structure of the big movie card using dummy data with the right positions according to the design.
Details: poster, average rating, the number of ratings, title, categories, description, director, and writer.
This is structure and will be worked later on to make pixel perfect.

# Jira task link: https://hackyourfuture-dk.atlassian.net/browse/VIEW-3

# Screenshots (if applicable):

Big Movie Card

### CODE REVIEW

File: packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ const TopCastDisplay = ({ movieID }) => {
         position: 'absolute',
         right: 280,
         left: 0,
+        top: 1300,
```
Comment (magdazelena) [inline packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx:20]: hmm, are we considering responsiveness in the project? Normally absolute positioning things is really ok, but it is safer in "small scale", so when I see large numbers I get a bit cautious :)

File: packages/client/src/components/BigMovieCard/BigMovieCard.jsx [hunk 2]
```diff
@@ -1,13 +1,156 @@
...
+        alignItems: 'center',
+        border: 1,
+        width: 1518,
+        height: 962,
+        paddingTop: 5,
+        paddingBottom: 5,
+        position: 'absolute',
+        left: 260,
+        top: 260,
+      }}
```
Comment (magdazelena) [inline packages/client/src/components/BigMovieCard/BigMovieCard.jsx:33]: I mentioned this in couple other PRs, but most of the components should be moved to some internal component library so you have one source of truth in most styling. You might want to discuss that in teams :)

For sure colors should be sourced in one file with constants, even if your colors are black, white and green. This makes it easier to make consistent changes across the project.

File: packages/client/src/components/RatingStars/RatingStars.jsx [hunk 3]
```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import Rating from '@mui/material/Rating';
 import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
 
 export default function RatingStars(params) {
```
Comment (magdazelena) [inline packages/client/src/components/RatingStars/RatingStars.jsx:6]: params doesn't seem to be in use here, does it?

File: packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx (hunk 1 above)
Comment (Aditi-Sawardekar) [reply packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx:20]: This is done temporarily to adjust position of the card on the Movie Details Page.
Our team will be working again on the positions of the components.

File: packages/client/src/components/RatingStars/RatingStars.jsx (hunk 3 above)
Comment (Aditi-Sawardekar) [reply packages/client/src/components/RatingStars/RatingStars.jsx:6]: Removed params as not used currently.

**Review by magdazelena — APPROVED** [review]

**Pull request merged**